temp.java

```diff
@@ -0,0 +1,109 @@
+import java.util.Scanner;
```

Comment:
temp.java should not have been added as part of program

Reply:
Yes I know, I accidentally left it there and it was too late as I had already submited it.